app/dts/behaviors/hid.dtsi

```diff
@@ -0,0 +1,9 @@
+/ {
+	behaviors {
+		hid_behavior: behavior_hid {
+			compatible = "zmk,behavior-hid", "zmk,behavior-global";
```

Comment:
Here is a global behavior. It listens for events and propagates HID updates to hosts (and eventually hubs from a spoke)